assets/app/view/create_game.rb

```diff
@@ -100,7 +100,8 @@ def render_content
           h(:a, { attrs: { href: '/login' } }, 'login'), ' to play multiplayer.'
         ]
       end
-      description << h(:p, 'If you are new to 18xx games then Shikoku 1889, 18Chesapeake or 18MS are good games to begin with.')
+      description << h(:p, 'New? Shikoku 1889 and 18Tokaido have Beginner variants '\
+                           'while 18Chesapeake and 18MS are good learning games')
```

Comment:
This seems to imply 1889 and 18Tokaido are not otherwise good learning games (I think they are).  If I'm making this change I'd just list out those four and leave it at that, if there's a beginner variant they can select it (or not).